apps/playground/src/app/modules/components/select-module/select-module.component.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Angular imports
+ */
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { TagType } from '@betagouv/ngx-dsfr/tag';
+import { SelectOption } from '@betagouv/ngx-dsfr/select';
+
+interface SettingsForm {
+  label: FormControl<string>;
+  description: FormControl<string>;
+  hint: FormControl<string>;
+  disabled: FormControl<boolean>;
+  multiple: FormControl<boolean>;
+  showSelectedOptions: FormControl<boolean>
+  hasFailed: FormControl<boolean>;
+  failureMessage: FormControl<string>;
+  hasSucceeded: FormControl<boolean>;
+  successMessage: FormControl<string>;
+}
+
+interface SimulationForm {
+  select: FormControl<string[]>;
+}
```

Comment:
I get that you want to copy/paste to save some time but please 🙏 ... 
I already told you guys we're not handling Typed Forms like this anymore...

No need for these interfaces, you can remove them.